Failing MongoDBConnectorCappedCollectionConsumerTest

The build on Friday March 27 failed with:

[...truncated 909.42 KB...] 
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.36 s - in io.syndesis.connector.mongo.MongoDBConnectorCappedCollectionConsumerAllOptionsTest 
[INFO] Running io.syndesis.connector.mongo.MongoDBConnectorCappedCollectionConsumerTest 
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.434 s <<< FAILURE! - in io.syndesis.connector.mongo.MongoDBConnectorCappedCollectionConsumerTest 
[ERROR] repeatedMongoTest(io.syndesis.connector.mongo.MongoDBConnectorCappedCollectionConsumerTest)  Time elapsed: 2.287 s  <<< FAILURE! 
java.lang.AssertionError: mock://result Received message count. Expected: <1> but was: <2> 
        at io.syndesis.connector.mongo.MongoDBConnectorCappedCollectionConsumerTest.mongoTest(MongoDBConnectorCappedCollectionConsumerTest.java:88) 
        at io.syndesis.connector.mongo.MongoDBConnectorCappedCollectionConsumerTest.repeatedMongoTest(MongoDBConnectorCappedCollectionConsumerTest.java:98)

--- a/app/connector/mongodb/src/test/java/io/syndesis/connector/mongo/embedded/EmbedMongoConfiguration.java
+++ b/app/connector/mongodb/src/test/java/io/syndesis/connector/mongo/embedded/EmbedMongoConfiguration.java
@@ -171,6 +171,7 @@ public class EmbedMongoConfiguration {
                 .net(new Net(PORT, Network.localhostIsIPv6()))
                 .replication(new Storage(storagePath.toString(),
                     REPLICA_SET, 5000))
+                .stopTimeoutInMillis(10000)
                 .build();
         } catch (IOException e) {
             throw new UncheckedIOException(e);

